lessons/models.py

```diff
@@ -47,6 +50,31 @@
 allowed_styles = ['background-color', 'color', 'font-family', 'font-size', 'font-style', 'font-width',
                   'strike-through', 'text-align', 'text-decoration']
 
+
+"""
+Keyword
+
+"""
+
+
+class Keyword(BaseKeyword, Internationalizable):
```

Comment:
I'd love to see some extensive comments explaining both what this model and what the `translated_keywords` methods are for, and why they're necessary